posthog/management/commands/test_migrations_are_safe.py

```diff
@@ -1,4 +1,5 @@
 import re
```

Comment:
Ignore changes in this file, I'll merge https://github.com/PostHog/posthog/pull/15277 first